docs/rst/recording/usage/configuration.rst

```diff
@@ -634,6 +645,13 @@ A complete example of all the configurations described on this page can be found
         max-rx-rate: 20
         downsampling: 3
 
+      logging:
+        verbosity: info
+        filter:
+          error: "DDSRECORDER"
+          warning: "DDSRECORDER"
+          info: "DDSPIPE|DDSRECORDER"
```

Comment:
I find it a bit weird to set as an example a configuration that would filter out more the errors and the warnings than the infos.